src/modules/admin/components/forms/UpdateFormDefinitionPage.tsx

```diff
@@ -4,40 +4,40 @@ import FormEditor from './FormEditor';
 import Loading from '@/components/elements/Loading';
 import PageTitle from '@/components/layout/PageTitle';
 import {
-  useGetFormDefinitionForEditorQuery,
+  useGetFormIdentifierForEditorQuery,
```

Comment:
I'm not convinced on whether this change makes sense. Let's discuss at check-in @martha. In my mind the "FormBuilder" page should still be working off the specific `FormDefinition` version that it's editing, so we could leave this query as it was.

It may need some more thought before implementing https://github.com/open-path/Green-River/issues/5858
For example, it brings up the question for me:
* When you hit "edit" on a form that doesn't currently have a draft, should it create a draft immediately and _then_ take you to editing that draft? Or, should it take you to a "new draft" page, and only generate the draft record when it auto-saves for the first time, and then going forward future auto-saves will update that draft? (Second option feels more complex imo)

It's kind of out of scope for this PR, which is why I think maybe leaving this page query as-is seems like a good idea.

Reply:
@gigxz would like to hear your thoughts about the new routing I implemented here, which now uses `forms/:identifier/:formId/edit` for the editor path.

It would be totally possible to just revert back to use `forms/:formId/edit`, but, then the breadcrumbs give a less-nice experience (or perhaps they just need more custom configuration to work nicely?)

Reply:
That seems fine to me. There is a way to update the breadcrumbs, I think, but it doesn't seem worth the hassle.